serverlessworkflow/modules/ROOT/pages/tooling/quarkus-dev-ui-extension/quarkus-dev-ui-workflow-definition-page.adoc

```diff
@@ -1,54 +1,110 @@
+[[con-start-workflow-instances-rest]]
+=== Starting new Workflow instances using REST
+If you want to start a new workflow instance using the workflow REST endpoint, just click on the *Start new Workflow*
+button of any of the workflow in the *Workflow Definitions* table, then you'll be redirected to the *Start New Workflow*
+page where you could setup the data and Business Key that will be used to start the new workflow instance.
 
-*If there is no JSON schema for the workflow, then the workflow is started by triggering a cloud event.
-To trigger a cloud event, you can use the *Cloud Event Type* and *Cloud Event Data* input fields to set the type of a cloud event and event payload in JSON format respectively and click the *Start* button to start a workflow. The *Reset* button on the page resets the values entered in the given fields.
+=== Filling up the Workflow data
+Depending on your workflow configuration the page can provide different mechanisms to fill the workflow data.
+If your workflow is configured with a valid _Data Input Schema_, this page will be able to display a custom Form to help
+a filling the data.
 
-.Starting a workflow using a cloud event
-image::tooling/quarkus-dev-ui-extension/kogito-swf-tools-trigger-cloud-events.png[]
+.Start Workflow Page with a Form defined in the _Data Input Schema_
+image::tooling/quarkus-dev-ui-extension/kogito-swf-tools-start-workflow-form.png[]
+
+If your workflow does not provide a _Data Input Schema_, the page will display a Code Editor that will enable you to manually fill the workflow data in JSON format.
+manually fill the workflow data in JSON format.
 
-If there is a JSON schema for the workflow configured in the `dataInputSchema` property, then a form is displayed to start a new workflow instance. You can fill in the required form details and click the *Start* button to trigger the workflow.
-The *Reset* button is used to clear the form data.
+.Start Workflow Page with a Code Editor.
+image::tooling/quarkus-dev-ui-extension/kogito-swf-tools-start-workflow-no-form.png[]
 
-.Starting a workflow using the form
-image::tooling/quarkus-dev-ui-extension/kogito-swf-tools-trigger-cloud-events-custom-form.png[]
+[NOTE]
+====
+For more information about how to setup the Input Schema Definition on your {context}, please take a look at the
+xref:core/defining-an-input-schema-for-workflows.adoc[Input Schema for {context}] section..
+====
 
-You can also use the *Business key* text box to define a custom business key value to the workflow instance. If the *Business Key* field is blank, then an auto-generated business key is defined for the workflow instance.
+=== Setting up the Business Key
+You can use the *Business key* text box to define a custom business key value to the workflow instance.
```

Comment:
maybe give more details about what business key is what for what it is used for?